CHANGES.rst

```diff
@@ -1,3 +1,564 @@
+  (which is only an issue when ERFA's built-in table is out of date). [#11234]
+
+- Fixed converting a zero-length time object from UTC to
+  UT1 when an empty array is passed. [#11516]
+
+astropy.uncertainty
+^^^^^^^^^^^^^^^^^^^
+
+- ``Distribution`` instances can now be used as input to ``Quantity`` to
+  initialize ``QuantityDistribution``.  Hence, ``distribution * unit``
+  and ``distribution << unit`` will work too. [#11210]
+
+astropy.units
+^^^^^^^^^^^^^
+
+- Move non-astronomy units from astrophys.py to a new misc.py file. [#11142]
+
+- The physical type of ``astropy.units.mol / astropy.units.m ** 3`` is now
+  defined as molar concentration.  It was previously incorrectly defined
+  as molar volume. [#11204]
+
+- Make ufunc helper lookup thread-safe. [#11226]
+
+- Make ``Unit`` string parsing (as well as ``Angle`` parsing) thread-safe. [#11227]
+
+- Decorator ``astropy.units.decorators.quantity_input`` now only evaluates
+  return type annotations based on ``UnitBase`` or ``FunctionUnitBase`` types.
+  Other annotations are skipped over and are not attempted to convert to the
+  correct type. [#11506]
+
+astropy.utils
+^^^^^^^^^^^^^
+
+- Make ``lazyproperty`` and ``classdecorator`` thread-safe. This should fix a
+  number of thread safety issues. [#11224]
+
+astropy.visualization
+^^^^^^^^^^^^^^^^^^^^^
+
+- Fixed a bug that resulted in some parts of grid lines being visible when they
+  should have been hidden. [#11380]
+
+- Fixed a bug that resulted in ``time_support()`` failing for intervals of
+  a few months if one of the ticks was the month December. [#11615]
+
+astropy.wcs
+^^^^^^^^^^^
+
+- ``fit_wcs_from_points`` now produces a WCS with integer ``NAXIXn``
+  values. [#10865]
+
+- Updated bundled version of ``WCSLIB`` to v7.4, fixing a bug that caused
+  the coefficients of the TPD distortion function to not be written to the
+  header. [#11260]
+
+- Fixed a bug in assigning type when converting ``colsel`` to
+  ``numpy.ndarray``. [#11431]
+
+- Added ``WCSCOMPARE_*`` constants to the list of WCSLIB constants
+  available/exposed through the ``astropy.wcs`` module. [#11647]
+
+- Fix a bug that caused APE 14 WCS transformations for FITS WCS with ZOPT, BETA,
+  VELO, VOPT, or VRAD CTYPE to not work correctly. [#11781]
+
+
+Other Changes and Additions
+---------------------------
+
+- The configuration file is no longer created by default when importing astropy
+  and its existence is no longer required. Affiliated packages should update their
+  ``__init__.py`` module to remove the block using ``update_default_config`` and
+  ``ConfigurationDefaultMissingWarning``. [#10877]
+
+- Officially declared minversion of ``ipython`` to be 4.2. [#10942]
+
+- Replace ``pkg_resources`` (from setuptools) with ``importlib.metadata`` which
+  comes from the stdlib, except for Python 3.7 where the backport package is added
+  as a new dependency. [#11091]
+
+- Turn on `numpydoc <https://numpydoc.readthedocs.io/en/latest/install.html>`__
```

Comment:
drop the numpydoc link, it would and will eventually break and doesn't bring much info into the changelog (which traditionally thus doesn't have links...)